### PR TITLE
Start fawiki wp10 model

### PR DESCRIPTION
**Pull request by Ladsgroup**
Bug: https://phabricator.wikimedia.org/T190050

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -112,6 +114,80 @@ enwiki_tuning_reports: \
 	tuning_reports/enwiki.wp10.md \
 	tuning_reports/enwiki.nettrom_wp10.md
 
+########################## Persian Wikipedia ###################################
+
+# https://quarry.wmflabs.org/query/25457
+datasets/fawiki.sampled_revisions.300.json:
+	wget https://quarry.wmflabs.org/run/244123/output/0/json-lines?download=true -qO- > $@
+
+datasets/fawiki.labelings.300.json:
```
Comment (halfak) [inline Makefile]: Maybe we should stick with "human_labeled" like in editquality

File: Makefile [hunk 2]
```diff
@@ -112,6 +114,80 @@ enwiki_tuning_reports: \
...
+	./utility fetch_labels \
+		https://labels.wmflabs.org/campaigns/fawiki/71/ > $@
```
Comment (halfak) [inline Makefile]: Should we also pull in the labels from campaign 70 (the pilot)?

File: Makefile [hunk 3]
```diff
@@ -112,6 +114,80 @@ enwiki_tuning_reports: \
...
+	  config/classifiers.params.yaml \
+	  wikiclass.feature_lists.fawiki.wp10 \
+	  wp10 \
+	  accuracy.macro \
+	  --pop-rate '"e"=0.7314705724717468' \
+	  --pop-rate '"bd"=0.2314879676843963' \
+	  --pop-rate '"b"=0.03023005873940185' \
+	  --pop-rate '"a"=0.0029374402333403227' \
+	  --pop-rate '"ba"=0.002439090897978488' \
+	  --pop-rate '"adq"=0.00143486997313615' \
```
Comment (halfak) [inline Makefile]: Looks like this is using frwiki classes.  Must be a WIP

File: Makefile (hunk 1 above)
Comment (Ladsgroup) [reply Makefile]: Done

File: Makefile (hunk 2 above)
Comment (Ladsgroup) [reply Makefile]: Yup, done

File: Makefile (hunk 3 above)
Comment (Ladsgroup) [reply Makefile]: Yes, On it

**Comment by Ladsgroup** [comment]
@halfak It was a miscalculation in number of stub articles. Fixed  now!

**Pull request merged**